libs/cloudsdk/cloudsdk.py

```diff
@@ -735,8 +742,8 @@ def upgrade_fw(self, equipment_id=None, force_upgrade=False, force_upload=False)
             try:
                 obj = self.equipment_gateway_client.request_firmware_update(equipment_id=equipment_id,
                                                                             firmware_version_id=firmware_id)
-                print("Request firmware upgrade Success! waiting for 100 sec")
-                time.sleep(100)
+                print("Request firmware upgrade Success! waiting for 300 sec")
```

Comment:
Future methods should concentrate on removing the sleeps

Reply:
I am worried if the cloudsdk method itself is self sufficient to verify the firmware upgrade. i wanna explore ucentral for that maybe